### PR TITLE
Patch to change install directories if a variable is set [6154]

### PR DESCRIPTION
**Pull request by AlvaroMolinillo**

### CODE REVIEW

File: foonathan_memory-config.cmake [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if(MSVC_VERSION EQUAL 1800)
+    if(CMAKE_CL_64)
+        include("${CMAKE_CURRENT_LIST_DIR}/../share/foonathan_memory-x64Win64VS2013/cmake/foonathan_memory-config.cmake")
```
Comment (richiware) [inline foonathan_memory-config.cmake]: Remove vs2013 arch.

File: CMakeLists.txt.patch [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ else()
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 030dba3..855a7aa 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -100,8 +100,14 @@ target_compile_definitions(foonathan_memory PUBLIC
+                             FOONATHAN_MEMORY_VERSION_MINOR=${FOONATHAN_MEMORY_VERSION_MINOR}
+                             FOONATHAN_MEMORY_VERSION_PATCH=${FOONATHAN_MEMORY_VERSION_PATCH})
+ 
++if(CMAKE_POSITION_INDEPENDENT_CODE)
```
Comment (richiware) [inline CMakeLists.txt.patch]: Better use BUILD_SHARED variable.

**Review by richiware — CHANGES_REQUESTED** [review]

File: CMakeLists.txt.patch (hunk 2 above)
Comment (AlvaroMolinillo) [reply CMakeLists.txt.patch]: Applied

File: foonathan_memory-config.cmake (hunk 1 above)
Comment (AlvaroMolinillo) [reply foonathan_memory-config.cmake]: Applied

Comment (AlvaroMolinillo) [reply foonathan_memory-config.cmake]: Applied

**Review by richiware — APPROVED** [review]

**Pull request merged**